core/src/com/unciv/ui/worldscreen/PlayerReadyScreen.kt

```diff
@@ -4,24 +4,113 @@ import com.badlogic.gdx.scenes.scene2d.Touchable
 import com.badlogic.gdx.scenes.scene2d.ui.Table
 import com.unciv.Constants
 import com.unciv.ui.images.ImageGetter
+import com.unciv.ui.popup.ToastPopup
 import com.unciv.ui.utils.BaseScreen
+import com.unciv.ui.utils.UncivTextField
 import com.unciv.ui.utils.extensions.onClick
 import com.unciv.ui.utils.extensions.toLabel
-import com.unciv.utils.concurrency.Concurrency
+import com.unciv.ui.utils.extensions.toTextButton
+import java.security.MessageDigest
 
 class PlayerReadyScreen(worldScreen: WorldScreen) : BaseScreen() {
+    var startTurn = true
```

Comment:
What does this actually mean? Unclear

Reply:
The entire screen has an onclick{} which changes the screen. I made this so I can disable that when clicking on the Textbox or on the buttons. Is there a cleaner way of doing this? I tried overriding the onClick but it still kept the one from the table.